src/Aws/DynamoDb/Streams/Types.hs

```diff
@@ -1092,15 +1096,84 @@ _StatusDisabled =
       fro = either pure (const $ pure StatusDisabled)
 {-# INLINE _StatusDisabled #-}
 
+data Stream
```

Comment:
Yes, ListStreamsResponse now gives a Stream, not just the identifier. You can also see that Stream is just a subset of the fields in StreamDescriptor:

https://docs.aws.amazon.com/dynamodbstreams/latest/APIReference/API_Stream.html